From: Joe Perches <joe@perches.com>
Date: Tue, 21 Dec 2010 02:16:08 -0800
Subject: [PATCH 30/47] drivers/net/*.c: Use static const

commit b6bc765067ece933cc3dc7f5e95665a89100b1d5 upstream.

Using static const generally increases object text and decreases data size.
It also generally decreases overall object size.

Signed-off-by: Joe Perches <joe@perches.com>
[bwh: Restricted to drivers/net/r8169.c]
---
 drivers/net/r8169.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/r8169.c b/drivers/net/r8169.c
index 50e7e5c..64bd0a4 100644
--- a/drivers/net/r8169.c
+++ b/drivers/net/r8169.c
@@ -1862,7 +1862,7 @@ static void rtl8168d_2_hw_phy_config(void __iomem *ioaddr)
 
 		val = mdio_read(ioaddr, 0x0d);
 		if ((val & 0x00ff) != 0x006c) {
-			u32 set[] = {
+			static const u32 set[] = {
 				0x0065, 0x0066, 0x0067, 0x0068,
 				0x0069, 0x006a, 0x006b, 0x006c
 			};
-- 
1.7.5.4

